src/components/forms/TextInput.jsx

```diff
@@ -0,0 +1,115 @@
+import React, { useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { throttle, debounce } from 'throttle-debounce';
+import IconInput from 'components/forms/IconInput';
+import DecoratedInput from 'components/forms/DecoratedInput';
+
+export const throttleValue = 500;
+
+const TextInput = ({
```

Comment:
Should we remove the stories for `IconInput` and `DecoratedInput` in this branch?

Reply:
I'm down, as long as you agree / see this component replacing `IconInput` / `DecoratedInput`.

Reply:
I'll take this question as an implicit thumbs up and go ahead and remove it for now 🙂

Reply:
Yep, I'd vote on removing those from docs